_posts/2021-06-25-app.md

```diff
@@ -0,0 +1,104 @@
+---
+layout: post
+title: "Unsere CargoRocket mobile App"
+author: David Prenninger
+excerpt: "Entwicklung einer multi-plattform Navigations-Applikation"
+description: "Entwicklung einer multi-plattform Navigations-Applikation"
+---
+
+Dieser Artikel soll Einblicke geben in die Idee und Entwicklung der ersten mobilen Lastenrad-Routing Applikation. Um mehr über den zugrundeliegenden Algorithmus zu erfahren lohnt sich ein Blick in unseren Artikel ["Der technische Aufbau unseres Lastenrad Routing"](https://cargorocket.de/2021/05/31/routing.html)
+Zum Ende des Artikels findet sich eine technische Beschreibung des Quellcodes der App. 
+
+## Motivation
+
+### Mobidata Hack
+Schon während des MobiData BW 2020 Hackathons, bei dem die Idee zum CargoRocket Lastenradrouting entstand, überlegten wir, wie wir das fertige Routing am besten präsentieren, validieren und am Ende auch verwendbar machen könnten. Schnell war klar, dass sich eine App für mobile Endgeräte sich besonders anbietet. Leider kam es während dem Hackathon zu [Technischen Schwierigkeiten](###Startschwierigkeiten), die uns daran hinderten schon zum Ende der 48h eine funktionsfähige App zu präsentieren. Als Ergebnis reichten wir als Routing-Demonstator eine Web-Applikation ein, die das Basisrouting auf eine einfache Weise visualisierte ([Github Repo [Archived]](https://github.com/CargoRocket/Launchpad)).
```

Comment:
ich würde gar nicht so auf die technischen Schwierigkeiten eingehen. Ich glaube kein Leser geht davon aus, dass man innerhalb von nicht mal 48h eine funktionsfähige App entwickelt ;)

VIelleicht eher sowas: "...sich besonders anbietet. Aufgrund der begrenzten Zeit, starten wir mit einem Routing-Demonstrator als Web-Applikation im Hackathon. In den letzten Monaten konnten wir die Anwendung als Mobile App entwickeln."

Reply:
Mh hab da 80% der Zeit in den App Prototypen gesteckt. Würde das gerne erwähnen. Ansich ist ein App Prototyp nicht aufwendiger als ein Webapp Prototyp.